### PR TITLE
Added "Flow over heated plate with two meshes" tutorial to doc.

### PR DESCRIPTION
**Pull request by boris-martin**
Updated the tutorial list & sidebar.
(Should be merged after that tutorial gets on the master branch of tutorials)

### CODE REVIEW

File: _data/sidebars/tutorial_sidebar.yml [hunk 1]
```diff
@@ -49,6 +49,10 @@ entries:
     - title: Flow over heated plate steady state
       url: /tutorials-flow-over-heated-plate-steady-state.html
       output: web
+    
+    - title: Flow over heated plate with two meshes
```
Comment (MakisH) [inline _data/sidebars/tutorial_sidebar.yml]: Shorter:

```suggestion
    - title: Flow over heated plate (two meshes)
```

But only it we also apply the same style to the "steady state" and "nearest projection" tutorials, actually.

**Review by MakisH — APPROVED** [review]
Nice, thanks!

I was wondering a bit about the order, but I guess the "nearest projection" and the "two meshes" are similarly complicated.

Please "squash and merge".

File: _data/sidebars/tutorial_sidebar.yml (hunk 1 above)
Comment (boris-martin) [reply _data/sidebars/tutorial_sidebar.yml]: Probably best yes, "Flow over heated plate steady state" without parentheses is ugly.

**Comment by boris-martin** [comment]
My intuition would be that two meshes is more complex than NP but simpler ("less different") than steady state. But do we want ascending order? Descending ?

(Yes I'll squash. No idea how all these old commits came up here)

**Review by MakisH — APPROVED** [review]
Looks good as it is! :+1:

**Pull request merged**